membership_withdrawal/views/membership_withdrawal_reason_view.xml

```diff
@@ -16,10 +16,22 @@
     <field name="arch" type="xml">
         <tree string="Membership withdrawal reasons" editable="top">
             <field name="name"/>
+            <field name="active" readonly="1" />
```

Comment:
I think this solution is ok, although I would prefer to have a form view defined for this model and a toggle button, to be consistent with the standard V12 archival feature.